cmd/snap/cmd_snapshot.go

```diff
@@ -469,3 +489,33 @@ func (x *exportSnapshotCmd) Execute([]string) (err error) {
 
 	return nil
 }
+
+type importSnapshotCmd struct {
+	clientMixin
+	durationMixin
+	Positional struct {
+		Filename string `long:"filename"`
+	} `positional-args:"yes" required:"yes"`
+}
+
+func (x *importSnapshotCmd) Execute([]string) error {
+	filename := x.Positional.Filename
+	f, err := os.Open(filename)
+	if err != nil {
+		return fmt.Errorf("error accessing file: %v", err)
+	}
+	defer f.Close()
+
+	importSet, err := x.client.SnapshotImport(f)
+	if err != nil {
+		return err
+	}
+
+	fmt.Fprintf(Stdout, "Imported snapshot #%d\n", importSet.ID)
```

Comment:
I would indicate that the imported snapshot got a new ID by saying:
```suggestion
	fmt.Fprintf(Stdout, "Imported snapshot as #%d\n", importSet.ID)
```